pkg/errors/errorMessage.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package errors
+
+type ErrorMessage struct {
+	Template string
+	Ordinal  int
+}
+
+var (
+	GALASA_ERROR_UNSUPPORTED_BOOTSTRAP_URL      = ErrorMessage{Template: "GAL1001E: unsupported bootstrap URL %s. Acceptable values start with 'http' or 'https'", Ordinal: 1001}
+	GALASA_ERROR_BOOTSTRAP_URL_BAD_ENDING       = ErrorMessage{Template: "GAL1002E: bootstrap url does not end in '/bootstrap'. Bootstrap url is %s", Ordinal: 1002}
+	GALASA_ERROR_BAD_BOOTSTRAP_CONTENT          = ErrorMessage{Template: "GAL1003E: bootstrap contents is badly formed. Bootstrap is at %s. Reason is: %s", Ordinal: 1003}
+	GALASA_ERROR_FAILED_TO_GET_BOOTSTRAP        = ErrorMessage{Template: "GAL1004E: failed to load the bootstrap from %s. Reason is %s", Ordinal: 1004}
```

Comment:
It's a picky point - but can we ensure consistent capitalisation across all the error messages please